server/src/organizations/organizations.module.ts

```diff
@@ -5,7 +5,8 @@ import { OrganizationsController } from './organizations.controller';
 import { Organization } from './entities/organization.entity';
 
 @Module({
-  imports: [TypeOrmModule.forFeature([Organization])],
+  imports: [
+    TypeOrmModule.forFeature([Organization])],
```

Comment:
also don't need this in your diff, not a change relevant to the scope of this PR

Reply:
copy!